src/services/chris_integration.tsx

```diff
@@ -238,119 +238,164 @@ class ChrisIntegration {
     return patientImages.fname;
   }
 
-  static async getPastAnalaysis(page: number, perpage: number): Promise<StudyInstanceWithSeries[]> {
+  /**
+   * Gets the ID of the latest Feed stored in Swift
+   */
+  static async getLatestFeedId(): Promise<number> {
+    const client: any = ChrisAPIClient.getClient();
+    const feeds = await client.getFeeds({
+      limit: 1,
+      offset: 0,
+    });
+    return feeds.getItems()?.[0]?.data?.id;
+  }
+
+  /**
+   * Starting at the provided offset, coninuously fetches Feeds from Swift until able to return an array of 
```

Comment:
Trailing whitespace at the end
```suggestion
   * Starting at the provided offset, coninuously fetches Feeds from Swift until able to return an array of
```